docs/legacy/getting-started-apm-server.asciidoc

```diff
@@ -229,64 +229,58 @@ IMPORTANT: {deprecation-notice-installation}
 [float]
 ==== Elastic Cloud
 
-If you're running APM Server in Elastic cloud, you can configure your own user settings right in the Elasticsearch Service Console.
-Any changes are automatically appended to the `apm-server.yml` configuration file for your instance.
-
-Full details are available in the {cloud}/ec-manage-apm-settings.html[APM user settings] documentation.
+If you're running APM Server in Elastic cloud, configure your user settings in the {ess} Console.
+See {cloud}/ec-manage-apm-settings.html[APM user settings] for more information.
 
 [float]
 ==== Self installation
 
-You can edit the `apm-server.yml` configuration file to customize it to your needs.
-The location of this file varies by platform, but the <<directory-layout>> will help you locate it.
-All available configuration options are outlined in
-{apm-server-ref-v}/configuring-howto-apm-server.html[configuring APM Server].
+Starting in version 8.0.0, {fleet} uses to the APM integration to set up and manage APM index templates,
+ILM policies, and ingest pipelines. APM Server will only connect to {es} _after_ the APM integration has been installed.
 
 [float]
-===== Standalone setup (data streams)
-
-If you're running APM Server as a standalone process, we recommend that you configure <<data_streams.enabled,`data_streams.enabled`>>
-to write events to data streams in the same way as the <<apm-integration>>. You will need to install the APM
-integration package to set up Elasticsearch before APM Server becomes operative.
+===== Install the APM integration
+
+. Open {kib} and select **Add integrations** > **Elastic APM**.
+. Click **APM integration**.
+. Click **Add Elastic APM**.
+. Click **Save and continue**.
+. Click **Add Elastic Agent later**. You do not need to run an {agent} to complete the setup.
+
+// Should we doc a workaround like this?
+// ----
+// curl localhost:5601/api/fleet/epm/packages/apm/{version} -X POST \
+// -H "Content-Type: application/json" \
+// -H "kbn-xsrf: kibana" \
+// -u elastic:changeme \
+// -d '{"force":true}'
+// ----
```

Comment:
Should we doc a workaround like this for installing the APM integration?

Reply:
Not sure I understand what the workaround is for. Why aren't the manual steps above enough?